modules/nf-core/svdb/merge/main.nf

```diff
@@ -48,7 +51,7 @@ process SVDB_MERGE {
     stub:
     def prefix = task.ext.prefix ?: "${meta.id}"
     """
-    touch ${prefix}.vcf.gz
+    echo | gzip > ${prefix}.vcf.gz
```

Comment:
why not just touch the file?

Reply:
1.12.4 Stub gzip files must use echo and pipe in https://nf-co.re/docs/guidelines/components/modules

>Simply touching a file with the file name ending in .gz will break nf-test’s Gzip file parser, as the file is not actually >gzipped and thus cannot be read.
>
>Therefore we must make sure we generate a valid gzipped file for nf-test to accept it during tests.